CONTRIBUTING.md

```diff
@@ -12,6 +12,20 @@ You can contribute in the following ways:
 
 If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
 
+## New to Open Source? START HERE
+
+We want all members of the GitHub community to feel welcome when it comes to contributing to our project. You can follow these step-by-step instructions to make a contribution via a pull request.
+
+1. Fork a repository from Mastodon and create a branch to build new features or test out ideas - instructions available here: https://help.github.com/en/articles/fork-a-repo
+2. Navigate to the original repository you created your fork from.
+3. To the right of the Branch menu, click New pull request.
+4. On the Compare page, click compare across forks.
+5. Confirm that the base fork is the repository you'd like to merge changes into. Use the base branch drop-down menu to select the branch of the upstream repository you'd like to merge changes into.
+6. Use the head fork drop-down menu to select your fork, then use the compare branch drop-down menu to select the branch you made your changes in.
```

Comment:
assuming someone knows what a branch is or how to make changes on it doesn't seem to make sense for this level of introductory guide?